cuml/kmeans_demo_mnmg.ipynb

```diff
@@ -95,21 +113,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "X_df, X_cudf = dask_make_blobs(n_samples, \n",
-    "                               n_features,\n",
-    "                               n_centers = 5, \n",
-    "                               n_parts = n_total_partitions,\n",
-    "                               cluster_std=0.1, \n",
-    "                               verbose=True)\n",
+    "X_cudf, Y_cudf = make_blobs(n_samples, \n",
```

Comment:
Lack of an X_df makes the sklearn example break.

We could just add "X_df = X_cudf.to_pandas()` and the same for y_cudf at the top of the sklearn example.

Reply:
Somehow I removed the `X_df` cell inadvertently. Just added it back.